frontend/src/components/landing/FaqAccordion.tsx

```diff
@@ -41,6 +41,13 @@ const QAndA = (props: {
 }) => {
   const buttonRef = useRef<HTMLButtonElement>(null);
 
+  useEffect(() => {
```

Comment:
I like that attention is given to the focus order :) Is that to prevent the user from accidentally following the link by pressing Enter?

One blocking concern I have, though, is that this applies everywhere the `<FaqAccordion>` is used, e.g. [on the landing page](https://deploy-preview-3386--fx-relay-demo.netlify.app/). So if you visit that now, the page will automatically scroll down to the FAQ.

Depending on what the goal is, I can think of two possible solutions. If it's just to avoid people from accidentally following the link right away, then maybe calling `.blur` on it on page load might solve that too? Otherwise, maybe we'd need an `autoFocus` prop on `<FaqAccordion>` that determines if this `.focus()` call runs.